rules/framework/framework_packaging.py

```diff
@@ -60,13 +60,18 @@ def _merge_binaries(framework_root, framework_name, binary_in):
         binary_in: a list of binary paths
     """
     output_path = os.path.join(framework_root, framework_name)
-    commands = ["libtool",
-                "-static",
-                "-D",
-                "-no_warning_for_no_symbols",
-                "-o", output_path
-                ] + binary_in
-    subprocess.check_call(commands)
+    if len(binary_in) == 1:
```

Comment:
do we have sample project with a single binary?

Reply:
anything pure objc will only have a single archive